lib/constants/audio-properties.js

```diff
@@ -0,0 +1,10 @@
+// constants
```

Comment:
Noticed that the partial transmuxer and the full transmuxer both use these. I can take this change out of this pull request if we want, but I kept it in for now as it was required at one point when I was adding tests for the partial transmuxer.